lib/xpi/utils.js

```diff
@@ -112,6 +104,35 @@ function createZip(options) {
   }
 
   return zip(options, dir, options.xpiPath).then(function(result) {
+    var fallbacks = options.fallbacks;
+
+    if (!fallbacks || (!fallbacks.installRDF && !fallbacks.bootstrapJS)) {
+      // return the zip file if there isn't any needed fallback file
+      return result;
+    }
+
+    return fs.readFile(options.xpiPath).then(function(data) {
+      // add the needed fallbacks files into the zip file
+      var zip = new Zip(data);
+
+      console.log("Inject bootstrap.js and install.rdf in " +
```

Comment:
was this just for debugging? or is it something that should be displayed while jpm is running?